Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedCollectionOperations.swift

```diff
@@ -75,6 +75,37 @@ struct UnifiedCreateIndex: UnifiedOperationProtocol {
     }
 }
 
+struct UnifiedListIndexes: UnifiedOperationProtocol {
```

Comment:
we don't actually end up using this operation since we skip the listIndexes cursor test, but I figured we would need it eventually and adding it seemed like a reasonable way to allow us to decode `cursors.json`